src/sentry/static/sentry/app/data/forms/organizationGeneralSettings.tsx

```diff
@@ -29,9 +28,8 @@ const formGroups: JsonFormObject[] = [
         name: 'name',
         type: 'string',
         required: true,
-
         label: t('Display Name'),
-        help: t('This is the name that users will see for the organization'),
+        help: t('A human-friendly name for the organization'),
```

Comment:
Hmm ... this is a little controversial because if the title is `Organization Slug`, I as a user would think that I should insert some name like "my-organization-name" and not "My organization's Name" .. maybe should we keep the label as "name" and add some information saying that this name will be used as an organization slug- AKA slugified?

Reply:
During organization creation, it will still be "name". It's only be "slug" in the settings, to avoid confusion such as [this case](https://forum.sentry.io/t/what-is-organization-slug-in-rest-api/3930).